librz/include/rz_asm.h

```diff
@@ -129,10 +130,9 @@ typedef struct rz_asm_plugin_t {
 	const char *cpus;
 	const char *desc;
 	const char *license;
-	void *user; // user data pointer
 	int bits;
 	int endian;
-	bool (*init)(void *user);
+	bool (*init)(void **user);
```

Comment:
I think init should also allow to pass some arguments (e.g. some plugins may require some information from the outside to configure special stuff). @thestr4ng3r cc

Reply:
There are NO plugins that requires special stuff.
if they needed something i would have seen since rzcore was passed as "user".
The only plugin that actually do need something is java, which requires to have access to RzBin to resolve class names, constant variables, etc..

Reply:
Right now there are not, but I remember discussing this with @thestr4ng3r not long ago. It was actually about IO and Bin, but the same logic applies i think.